src/game/Editor/SmartMethod.cc

```diff
@@ -34,8 +34,8 @@ void CalcSmartBrokenWallDefault( UINT16 *pusObjIndex, UINT16 *pusUseIndex )
 	switch( gubBrokenWallUIValue )
 	{
 		case 0:case 1: *pusUseIndex = 49 + gubBrokenWallUIValue;	break;
-		case 3:				 *pusUseIndex = 62;													break;
-		case 4:				 *pusUseIndex = 64;													break;
+		case 3:		*pusUseIndex = 62;				break;
+		case 4:		*pusUseIndex = 64;				break;
```

Comment:
not precisely whitespace and only a suggestion:
split case 0 and case 1 labels onto two lines, align the statements, extra line for break

off topic: what about case 2 here?